x-pack/docs/en/security/authentication/remote-clusters-privileges.asciidoc

```diff
@@ -0,0 +1,225 @@
+[[remote-clusters-privileges]]
+=== Configure remote cluster privileges
+After <<remote-clusters-connect,connecting to remote clusters>>, you create a
+user role on both the local and remote clusters and assign necessary privileges.
+These roles are required to use {ccr} and {ccs}.
+
+IMPORTANT: You must use the same role names on both the local
+and remote clusters. For example, the following configuration for {ccr} uses the
+`ccr_user` role name on both the local and remote clusters. However, you can
+specify different role definitions on each cluster.
+
+You can manage users and roles from Stack Management in {kib} by selecting
+*Security > Roles* from the side navigation. You can also use the
+<<security-role-mapping-apis,role management APIs>> to add, update, remove, and
+retrieve roles dynamically. When you use the APIs to manage roles in the
+`native` realm, the roles are stored in an internal {es} index.
+
+The following requests use the
+<<security-api-put-role,create or update roles API>>.
+
+
+[[remote-clusters-privileges-ccr]]
+//tag::configure-ccr-privileges[]
+==== Configure privileges for {ccr}
+The {ccr} user requires different cluster and index privileges on the remote
+cluster and local cluster. If necessary,
+<<security-api-put-user,create a user>> that you want to assign the {ccr} role
+to.
+
+[discrete]
+===== Remote cluster
+On the remote cluster that contains the leader index, the `ccr_user` requires
+the `read_ccr` cluster privilege, and `monitor` and `read` privileges on the
+leader index.
```

Comment:
I raised #61308 as a bug for CCR privileges a while back. Not sure if it is still valid today. I can check it later.